app/Mcp/Tools/ListServices.php

```
@@ -0,0 +1,68 @@
+    use BuildsResponse;
+    use ResolvesTeam;
+
+    public function handle(Request $request): Response
+    {
+        if ($error = $this->ensureAbility($request, 'read')) {
+            return $error;
+        }
+
+        $teamId = $this->resolveTeamId($request);
+        if (is_null($teamId)) {
+            return Response::error('Invalid token.');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Laravel\Mcp\Response' in method 'handle'. (undefined)

(StaticAccess)